zheng-common/pom.xml

```diff
@@ -341,8 +347,8 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.18.1</version>
                 <configuration>
-                    <skipTests>true</skipTests>
-                    <testFailureIgnore>true</testFailureIgnore>
+                    <skipTests>false</skipTests>
+                    <testFailureIgnore>false</testFailureIgnore>
```

Comment:
Is there a reason that they did this in the first place? Do they have any other tests in this module?

Reply:
They do not.